airbyte-integrations/connectors/source-open-exchange-rates/source_open_exchange_rates/source.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import pendulum
+from pendulum import DateTime
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class OpenExchangeRates(HttpStream, ABC):
+    url_base = "https://openexchangerates.org/api/"
+
+    primary_key = None
+    date_field_name = "timestamp"
+    cursor_field = date_field_name
```

Comment:
nitpick: we could just have one property here

```suggestion
    cursor_field = "timestamp"
```